sim_instr/CMakeLists.txt

```diff
@@ -0,0 +1,199 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(sim_instr)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++14)
```

Comment:
use set(CMAKE_CXX_STANDARD 14) instead for portability

also: set(CMAKE_CXX_STANDARD_REQUIRED ON) set(CMAKE_CXX_EXTENSIONS OFF)